tracker_lib.py

```diff
@@ -31,10 +37,11 @@ def nothing(a):
  
 #while not key == ord('q'):
 TH = cv2.getTrackbarPos('TH','threshold')        ### gustavo 
-upper_white = np.array([TH], dtype=np.uint8)  ### gustavo 
-arduino = serial.Serial('/dev/ttyACM0', 9600) ### gustavo
+upper_white = np.array([TH], dtype=np.uint8)  ### gustavo
+# arduino = serial.Serial('/dev/ttyACM0', 9600) ### gustavo
```

Comment:
This line must be un-commented. It was commented at the beginning only for debug purposes.